demos/serverless/deploy.js

```diff
@@ -11,8 +11,8 @@ let app = `meetingV2`;
 let useEventBridge = false;
 let enableTerminationProtection = false;
 let disablePrintingLogs = false;
-let chimeEndpoint = 'https://service.chime.aws.amazon.com';
-let chimeServicePrincipal = 'chime.amazonaws.com'
+let chimeEndpoint = 'https://meetings-chime.us-east-1.amazonaws.com ';
```

Comment:
Do we need to make this change? ChimeServicePrincipal is used by media capture i think.